doc/index.rst

```diff
@@ -2,14 +2,11 @@
 
 .. title:: Matplotlib: Python plotting
 
-Matplotlib is a Python 2D plotting library which produces publication quality
-figures in a variety of hardcopy formats and interactive environments across
-platforms.  Matplotlib can be used in Python scripts, the Python and IPython_
-shells, the Jupyter_ notebook, web application servers, and four graphical user
-interface toolkits.
+Matplotlib: 2D visualization in Python
+--------------------------------------
 
-.. _IPython: http://ipython.org
-.. _Jupyter: http://jupyter.org
+Matplotlib supports your visualization tasks, from simple one-off plotting to
+complex and interactive figures.
```

Comment:
I think this is a very good summary of what Matplotlib does, but I would love to see the stress on "publication quality" figures in this one line summary.
It's all going to be about tradeoff, so this comment should (of course) not hold this PR. Here's a suggestion that builds on the work you've already done.

- Matplotlib supports your visualization tasks, from simple one-off plotting to complex, publication quality, and interactive figures.

Reply:
Thanks for the feedback. I've added your suggestion.

Reply:
Thanks!